kubecost/values.yaml

```
@@ -813,6 +813,20 @@ networkCosts:
       #      - "15.128.15.2"
       #      - "20.0.0.0/8"
 
+  ## Network costs is a daemonset and may need to be scheduled on nodes with specific taints.
+  ## If monitoring the network costs for workloads on nodes with specific taints, add the tolerations here.
+  ## to tolerate all taints, use the following:
+  ## tolerations:
+  ##   - operator: Exists
+  ##     effect: NoSchedule
+  ## to tolerate specific taints, use the following:
+  ## tolerations:
+  ##   - operator: Exists
+  ##     effect: NoSchedule
+  ##     key: "key"
+  ##     value: "value"
+  ##     operator: Equal
+  ##     effect: NoSchedule
```

There are duplicate `operator` and `effect` here. I think if you're going to include both a key and a value you need to use `operator: Equal`? ([docs ref](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/#concepts))

```suggestion
  ##   - operator: Equal
  ##     effect: NoSchedule
  ##     key: "key"
  ##     value: "value"
```